region/brinstar/blue/Blue Brinstar Energy Tank Room.json

```diff
@@ -706,6 +706,31 @@
       "clearsObstacles": ["A", "B", "C"],
       "note": "Placing the bomb against the bomb block at the top of the shaft can be somewhat tricky."
     },
+    {
+      "link": [4, 2],
+      "name": "Geemer Ice Climb",
+      "requires": [
+        {"or": [
+          "h_canUsePowerBombs",
+          {"and": [
+            {"obstaclesCleared": ["B"]},
+            "Morph"
+          ]},
+          {"obstaclesCleared": ["C"]}
+        ]},
+        {"or": [
+          {"obstaclesCleared": ["A"]},
+          "ScrewAttack",
```

Comment:
This felt slow to me, should it require patience? It can be done with a PB and then going back for the non-global geemer. Although, that one is harder to climb, and may even require very patient. The blind part is more challenging with it as well. (Can't use the non-global with B unless you have morph, even tho there's no reason to do this strat with B without morph.)

With bombs, you can just IBJ, but it's fine to keep it in here.